### ISSUE TITLE
Registry tool: setting unknown `registry config` parameters seems to silently fail

### ISSUE THREAD

**Issue by timburks** [apigee/registry#1096]
`invalid` here could be any unrecognized parameter.

```
% registry config set invalid parameter
----
Need more help?
https://github.com/apigee/registry/wiki

% registry config set invalid 123      
----
Need more help?
https://github.com/apigee/registry/wiki
```

**Comment by theganyo** [comment]
Interesting. This was broken by https://github.com/apigee/registry/pull/915.

**Comment by theganyo** [comment]
I'm going to remove the:

```
cmd.SilenceUsage = true
cmd.SilenceErrors = true
```

flags from all the configuration commands. IIRC, these were there to avoid the issue that we solved in #915 without breaking the other commands.
